modules/ROOT/pages/_partials/generic-local-mobile-dev-envir-reqs.adoc

```diff
@@ -29,7 +29,7 @@ If you have not installed CocoaPods, install it by running the command:
 $ gem install cocoapods
 ----
 
-For information on how to set up your local development environment for iOS, see link:https://developer.apple.com/library/archive/referencelibrary/GettingStarted/DevelopiOSAppsSwift/index.html[iOS, window="_blank"].
+For information on how to set up a local iOS development environment, see link:https://developer.apple.com/library/archive/referencelibrary/GettingStarted/DevelopiOSAppsSwift/index.html[iOS, window="_blank"].
```

Comment:
see the link:https://developer.apple.com/library/archive/referencelibrary/GettingStarted/DevelopiOSAppsSwift/index.html[Swift documentation, window="_blank"].